### DISCUSSION TITLE
Open modal or action on select

### DISCUSSION THREAD

**Discussion by Dontorpedo** [filamentphp/filament#5250] [General]
Hello,

How can i open a modal or fire a action after a select field is changed?

thanks

**Comment by danharrin** [comment]
`->reactive()->afterStateUpdated(fn ($state) => runs after select changes)`

**Reply by Dontorpedo** [reply]
thanks, worked,.. any way to open a modal after a model i created?

**Reply by danharrin** [reply]
You'd have to emit a Livewire event or something and open a modal from that... but theres nothing built into Filament to do that

**Comment by Dontorpedo** [comment]
ah right, thanks..
